src/tables.ts

```diff
@@ -41,7 +41,6 @@ export const CommonMetrics: Record<string, any> = {
   },
 
   FretHandFinger: {
-    fontFamily: 'Arial, sans-serif',
```

Comment:
Since fontFamily is not specified here, does that mean that it will default to whatever "text font" we are using? It looks like the new images are using Roboto Slab?

Reply:
They default to the one set in the HACK where you placed the HACK begin / end.